### DISCUSSION TITLE
Fehler: Service pyscript.set_ef_powerstream_custom_load_power not found

### DISCUSSION THREAD

**Discussion by Quadrophon** [svenerbe/ecoflow_dynamic_power_adjustment#17] [General]
Hallo zusammen,

versuche jetzt schon einige Zeit inkl. einiger neu Installationen von pyscript das Script zum laufen zu bringen, leider ohne Erfolg. Ich bekomme jedes mal bei Ausführung meiner Automation die Fehlermeldung:

**Fehler: Service pyscript.set_ef_powerstream_custom_load_power not found**

Meine Automation sieht so aus:

service: pyscript.set_ef_powerstream_custom_load_power
data:
  SerialNumber: SERIAL-NUMBER
  TotalPower: "{{ states('sensor.shellypro3em_SERIAL-NUMBER_total_active_power') | round }}"
  Automation: true

Das Script liegt unter /homeassistant/pyscript/. Habe aber auch schon andere Ablageorte probiert, z.B. 

/homeassistant/custom_components/pyscript/
/homeassistant/config/pyscript/

Alles leider ohne Erfolg.

Hat evtl. jemand eine Lösung für mein Problem. 
Vielen Dank im voraus.

**Comment by svenerbe** [comment]
Hi, hast du mal geprüft ob du den service unter entwickerwerkzeuge findest?

![image](https://github.com/svenerbe/ecoflow_dynamic_power_adjustment/assets/24878253/f8693f9b-6b77-45b2-bbf9-50dcdc897df5)

**Reply by Quadrophon** [reply]
Hallo, ja das habe ich. Dort tauchen leider nur die beiden Dienste:

![Bildschirmfoto 2024-05-20 um 17 07 02 PM](https://github.com/svenerbe/ecoflow_dynamic_power_adjustment/assets/127607094/4be26c1b-5a28-4c2c-b74c-459eb3f30e11)



auf. Bin da gerade echt am verzweifeln.

**Comment by Stuboeys** [comment]
I am facing the same error.
The log suggests that is has something to do with `import sys`

> Logger: custom_components.pyscript.file.set_ef_powerstream_custom_load_power
Quelle: custom_components/pyscript/global_ctx.py:341
Integration: Pyscript Python scripting ([Dokumentation](https://github.com/custom-components/pyscript), [Probleme](https://github.com/custom-components/pyscript/issues))
Exception in </config/pyscript/set_ef_powerstream_custom_load_power.py> line 16: import sys ^ ModuleNotFoundError: import of sys not allowed

**Comment by svenerbe** [comment]
https://github.com/custom-components/pyscript

have you check section configuration?

**Reply by Stuboeys** [reply]
My issue was apparently indeed related to the general configuration. Works now. Thanks!
